tests/php/tests/includes/test_class.wp-job-manager-email-notifications.php

```diff
@@ -429,10 +429,15 @@ public function test_add_email_settings() {
 				'enabled'    => $email_is_default_enabled ? '1' : '0',
 				'plain_text' => '0',
 			];
+			$email_settings_count = count( $email_settings ) + 1;
+
+			if ( in_array( $setting['name'], [ 'job_manager_email_admin_updated_job', 'job_manager_email_admin_new_job', 'job_manager_email_admin_expiring_job', 'job_manager_email_valid_email' ] ) ) {
+				$email_settings_count++;
+				$defaults = array_merge( array_splice( $defaults, 0, 1 ), [ 'email_to' => '' ], $defaults );
```

Comment:
I am not sure what this line tries to achieve here. Can we write this as
```
$defaults['email_to'] = '';
```

Reply:
Good question!  It has to be in a specific order for the PHPUnit tests to not fail because the unit tests expect the arrays to be exactly the same - so this is just my way of making the arrays identical.